mmeds/spawn.py

```diff
@@ -217,6 +219,18 @@ def log_processes(self):
         # Remove logged processes so they aren't recorded twice
         self.processes.clear()
 
+
+    def clean_temp_folders(self):
+        """ Clean out temp folders older than a day, once every day."""
+        if datetime.utcnow() - self.cleaned_temp > timedelta(days=1):
+            temp_files = (Path(fig.DATABASE_DIR) / 'temp_dir').glob('*')
+            for temp_file in temp_files:
```

Comment:
Are these temp_files or temp_dirs? Going based on the age of the directory is probably fine and definitely more efficient but possibly could cause problems for very long uploads. At this point it might just be worth noting that possibility somewhere

Reply:
they are temp dirs, wouldn't the upload need to take longer than a day to be an issue? seemed like a safe enough margin for me.

I'll update the name from temp_files to temp_sub_dirs